### ISSUE TITLE
sam local invoke Fn

### ISSUE THREAD

**Issue by vespertilian** [studds/nx-aws#58]
I think we should support local invoke for simple functions. I currently have a use case where I want to test some plain functions it's a bit of a pain. Unless you can already do this and I just don't know how.

**Comment by studds** [comment]
I agree this would be useful. Currently the way to do it is to run the build & then invoke the template in the dist directory directly. Part of the reason _not_ to do this is that if you were testing something locally, you'd probably want to run a build server in watch mode ie `nx build api --watch` and then invoke on demand. That kind of flow would be a little difficult to add in a robust way....

Unless of course you spun up the build server and then added a prompt for the user to select which function they'd want to invoke? That would be cool and super friendly, but much more work.

There's also the question of how to select the payload.

Thoughts welcome!
